[BUG] @ng-web-apis module can't be resolved in CI pippeline testing

# 🐞 Bug report

### Description

Micro-Sentry versions starting with 3.0.0 break ng test in a CI pipeline with the following error:
./node_modules/@micro-sentry/angular/fesm2020/micro-sentry-angular.mjs:9:0-45 - Error: Module not found: Error: Can't resolve '@ng-web-apis/common' in '/builds/glitchtip/glitchtip-frontend/node_modules/@micro-sentry/angular/fesm2020' .

### Reproduction

I have only been able to reproduce this in the CI pipeline. It does not happen locally and only appears to impact testing.

### Expected behavior

Tests shouldn't break and the @ng-web-apis peer dependency should be handled by micro-sentry.

### Versions

@micro-sentry V.3.0.0 and above

If needed:

Angular 14

### Additional context

<!-- Add any other context about the problem here -->

 As an experiment, I was able to avoid this error by adding @ng-web-apis/common as a dependency on my own project. But I don't think that should be necessary. 
 
 I set up a repository for testing. You can see the failed pipeline here:
 https://gitlab.com/james.kiger/micro-sentry-test/-/jobs/2576177389

--- a/libs/angular/ng-package.json
+++ b/libs/angular/ng-package.json
@@ -4,5 +4,5 @@
   "lib": {
     "entryFile": "src/index.ts"
   },
-  "allowedNonPeerDependencies": ["@micro-sentry/browser"]
+  "allowedNonPeerDependencies": ["@micro-sentry/browser", "@ng-web-apis/common"]
 }

--- a/libs/angular/package.json
+++ b/libs/angular/package.json
@@ -3,11 +3,11 @@
   "version": "0.0.0-development",
   "peerDependencies": {
     "@angular/common": "^14.0.0",
-    "@angular/core": "^14.0.0",
-    "@ng-web-apis/common": "^1.12.0"
+    "@angular/core": "^14.0.0"
   },
   "dependencies": {
     "tslib": "^2.0.0",
+    "@ng-web-apis/common": "^1.12.0",
     "@micro-sentry/browser": "0.0.0-development"
   },
   "license": "Apache-2.0",

